tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -949,6 +968,20 @@ assign_fortran_storage_classes(void)
   }
 } /* end assign_fortran_storage_classes() */
 
+/*
+ * when vector always pragma is specified, "llvm.loop.parallel_accesses" metadata has
+ * to be generated along with "llvm.access.group" for each load/store instructions.
+ */
+INLINE static LL_MDRef
+cons_loop_parallel_accesses_metadata(void)
+{
+  LL_MDRef lvcomp[2];
+
+  lvcomp[0] = ll_get_md_string(cpu_llvm_module, "llvm.loop.parallel_accesses");
+  lvcomp[1] = cached_access_group_metadata;
```

Comment:
Is it OK to reuse the access-group-metadata always?

Reply:
Yes it should be fine. Even if for example we have multiple functions with different loops using this pragma, the metadata is actually replicated for each set of the `llvm.access.group` (therefore multiple instances of `llvm.loop.parallel_accesses` metadata is generated).